.github/issue_template.md

```diff
@@ -0,0 +1,17 @@
+**Is this a BUG REPORT or FEATURE REQUEST?**:
+
+- [ ] bug
+- [ ] feature
```

Comment:
IHMO, it's better to avoid using `checkbox` here. Because this checkbox is treated as tasks which are required to complete this issue according to [GitHub's manner](https://help.github.com/articles/about-task-lists/).

What do you think? Don't you mind such manner?